doc/man1/openssl-ocsp.pod.in

```diff
@@ -167,6 +169,23 @@ I<hostname> on port I<port>. The B<-path> option specifies the HTTP pathname
 to use or "/" by default.  This is equivalent to specifying B<-url> with scheme
 http:// and the given hostname, port, and pathname.
 
+=item B<-proxy> I<[http[s]://][userinfo@]host[:port][/path]>
+
+The HTTP(S) proxy server to use for reaching the OCSP server unless B<-no_proxy>
+applies, see below.
+The proxy port defaults to 80 or 443 if the scheme is C<https>; apart from that
+the optional C<http://> or C<https://> prefix is ignored,
+as well as any userinfo and path components.
+Defaults to the environment variable C<http_proxy> if set, else C<HTTP_PROXY>
```

Comment:
you should update `doc/man7/openssl-env.pod`  And make it uppercase only.

Reply:
No, I would not. The feature to support both variants is already there and for a good reason.

Reply:
`doc/man7/openssl-env.pod` is about environment variables defined by OpenSSL.
`HTTP_PROXY` and friends have been defined elsewhere.